src/content/docs/alerts-applied-intelligence/new-relic-alerts/get-started/streaming-alerts-key-terms-concepts.mdx

```diff
@@ -32,6 +32,88 @@ Even if a data point meets the criteria for a violation, a violation isn't trigg
 
 All of these configurable delays give you more control over how you're alerted on sporadic and missing data.
 
+## Streaming alerts process and descriptions [#streaming-table]
+
+<table class="alternate">
+  <thead>
+    <tr>
+      <th style={{ width: "200px" }}>
+        Process
+      </th>
+
+      <th>
+        **Description**
+      </th>
+    </tr>
+  </thead>
+    <tbody>
+    <tr>
+      <td>
+        Streaming data
+      </td>
+      <td>
+        All data coming into New Relic.
+      </td>
+    </tr>
+    
+    <tr>
+      <td>
+        NRQL WHERE clause
+      </td>
+      <td>
+        Filters all incoming streaming data. We only monitor for alerts on data that makes it through this filter.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Aggregation window
+      </td>
+      <td>
+        A specific amount of time when the filtered data (or its absence) can accumulate in this window.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Evaluation offset
+      </td>
+      <td>
+        A time delay to ensure data points are placed in the correct aggregation window.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Aggregated data
+      </td>
+      <td>
+        Data in the aggregated window is collapsed to a single data point for alert evaluation.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Evaluation
+      </td>
+      <td>
+        The data point is evaluated by the NRQL condition, which is trigged by each incoming aggregated data point.
+      </td>
+    </tr>
+
+    <tr>
+      <td>
+        Threshold duration
+      </td>
+      <td>
+        A specific duration that determines if a violation is created. If your specified NRQL condition meets the threshold criteria over the threshold duration, a violation occurs. 
+        
+        When a data point lacks data, a custom value is inserted to fill the gap. 
+      </td>
+    </tr>
+    </tbody>
+</table>
+
 ## Streaming alerts tools
```

Comment:
Add anchor.

Reply:
This is a good suggestion! However, I decided to add a header that we can link to because it seems to match more closely what we're doing throughout the site.